.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,57 @@
+name: "godot-ci export"
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+env:
+  GODOT_VERSION: 4.2.2
+  EXPORT_NAME: sequence-escape
+
+jobs:
+  export-web:
+    name: Web Export
+    runs-on: ubuntu-20.04
+    container:
+      image: barichello/godot-ci:4.2.2
```

Comment:
It does not seem to be possible to use `${{ env.GODOT_VERSION }}` here, but they do have to match.